sotodlib/toast/ops/load_context.py

```diff
@@ -367,7 +464,7 @@ def _exec(self, data, detectors=None, **kwargs):
 
             focalplane = toast.instrument.Focalplane(
                 detector_data=det_props,
-                sample_rate=rate,
+                sample_rate=1.0 * u.Hz,
```

Comment:
Is this a leftover debugging setting?

Reply:
No, this is an optimization.  When we create the observation, we do not yet have timestamps (only the number of samples).  So we cannot compute the sample rate yet without reading through the entire dataset (which is what was happening before in the code).  Instead, we set the focalplane to a fake sample rate and later (after the timestamps are loaded) we go back and fix the sample rate.